src/main/java/de/unistuttgart/overworldbackend/service/MinigameTaskStatisticService.java

```diff
@@ -139,6 +139,7 @@ private List<PlayerTaskActionLog> getOrderedPlayerTaskActionLogByDate(
         return successfulPlayerTaskStatistic
             .getPlayerTaskActionLogs()
             .stream()
+            .filter(e -> e.getDate() != null)
```

Comment:
In commit https://github.com/Gamify-IT/overworld-backend/pull/68/commits/19d0ed63db522cd2808332bdbb9d71bdeee9f4f8 i figured out, that the date value is sometimes null, because the annotation @CreationTimestamp is not working as excepted. I fixed it with using @PrePersist and set the date to current.